textproc/wbxml2/Portfile

```diff
@@ -1,33 +1,28 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        libwbxml libwbxml 0.11.8 libwbxml-
+github.tarball_from archive
 
 name                wbxml2
-version             0.11.6
 categories          textproc devel
 license             LGPL-2.1+
 platforms           darwin
```

Comment:
We can drop this line too, `platforms darwin` is now implicit in MacPorts.

Reply:
Cool, removed